api_spec.yaml

```diff
@@ -322,12 +323,18 @@ paths:
                       $ref: '#/components/schemas/createJWTPayload'
                   required:
                     - authorizationCode
+                    - clientId
                 - type: object
                   properties:
                     clientId:
                       type: string
+                      $ref: '#/components/schemas/oauth2ClientId'
                     clientSecret:
```

Comment:
status UNKNOWN_REFRESH_TOKEN_LIFE_TIME_ERROR is badly named - it makes no sense. What is this for?